packages/__docs__/buildScripts/DataTypes.mts

```diff
@@ -85,77 +92,61 @@ type JSDocFunctionReturns = {
     names: string[]
   }
 }
-// TODO remove these types, now we can get them directly from react-docgen
+// TODO these are from React-docgen Documentation.d.ts,
+// remove when react-docgen exports them
```

Comment:
I've opened a ticket for them https://github.com/reactjs/react-docgen/issues/864

Reply:
this code is just a refresh (copypaste) of their type definitions